configure.ac

```diff
@@ -170,6 +182,18 @@ AC_LINK_IFELSE(
 	])
 AC_LANG_POP(C)
 
+dnl ----------
+dnl Essentials
+dnl ----------
+
+AX_PTHREAD([
```

Comment:
Please see my comment on AX_PTHREAD in #302. Looks like builds succeed in CI with this, but they also succeed with AC_SEARCH_LIBS, without requiring setting PTHREAD_CC, CFLAGS, etc; I will defer to your expertise on which one is better.